proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +46,7 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  string owner_name = 10;
```

Comment:
I am a little concerned about the duplication here, but I recognize that this probably makes things easier both in the CLI and the frontend 👍🏻.